turbinia/workers/__init__.py

```diff
@@ -429,6 +431,18 @@ def run_wrapper(self, evidence):
       try:
         self.result = self.setup(evidence)
         original_result_id = self.result.id
+
+        if self.turbinia_version != turbinia.__version__:
+          msg = 'Worker V-{} and server V-{} versions do not match'.format(
```

Comment:
Can you change the format strings to be like {0:s}?  See https://github.com/google/turbinia/blob/master/docs/contributing.md#style-guide for more details.

Other than that, looks good, thanks!